app/views/group_offers/_form.html.slim

```diff
@@ -72,6 +62,30 @@
     .col-xs-12
       = f.input :schedule_details
 
+  h3= t_attr(:volunteers)
+  .row
+    .col-xs-12.col-md-4
+      = f.input :necessary_volunteers, collection: 1..10
+    - if @group_offer.volunteers.any?
+    .col-xs-12.col-md-4
+      =t_attr(:volunteers)
+      ul
+        - @group_offer.volunteers.each do |volunteer|
+          li= link_to volunteer, volunteer_path(volunteer)
+    .col-xs-12.col-md-4
+      button.btn.btn-primary(data-toggle='collapse' data-target='#volunteerBlock' type='button'  aria-expanded='false' aria-controls='volunteerBlock')
+        - if action_name ==  'new' || @group_offer.volunteers.empty?
+          = t('.add_volunteers')
+        - else
+          = t('.edit_volunteers')
+    #volunteerBlock.collapse.col-xs-12.col-md-4
+      - if policy(GroupOffer).superadmin?
```

Comment:
`.superadmin?` and `department_manager?` policies are not defined. To me there is no reason to change the `superadmin_priviledges?` policy. Also there is no reason to have to ifs executed when only one of the cases will be true each time.